src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
I know this file must have been generated automatically by some lib but can you give me a little bit of explanation what is it used for?

Reply:
What God has given you, don't touch it 😄 .
In our case, it did nothing since it was turned off.